portfolio2.css

```diff
@@ -0,0 +1,188 @@
+* {
+	scroll-behavior: smooth;
+    box-sizing: border-box;
+	margin: 0;
+	font-family: ubuntu, sans-serif;
+}
+
+body {
+    margin:0px;
+   }
+	
+  div.links {
+	    background-color:transparent;
+		border-bottom-style: solid;
+		border-bottom-color: Lightgrey;
+		padding: 10px;
+		display: flex;
+		position: absolute;
+		top: 1px;
+		 }
+		
+	.links a{
+		display:inline;
+		color: #7393B3;
+		line-height:1.5em;
+		padding: 9px 29px 24px 52px;
+		text-decoration:none;
+		height:90px;
+		font-size: 40px;
+		
+		
+	}
+	
+	.links a:hover{
+		color:#0047AB;
+		
+	}
+	
+	nav.container {
+		width:100%;
+	    background-image: url("https://images.pexels.com/photos/5044497/pexels-photo-5044497.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+		background-repeat:no-repeat;
+		background-size:cover;
+		display:flex;
+		padding-top: 693px;
+		padding-bottom: 188px;
+		justify-content:space-evenly;
+		position: relative;
+	}
+	
+	
+	div.aboutme {
+	    border: 2px transparent solid;
+	    border-top-color: black;
+	    border-bottom-color: black;
+	    background-color: #AEC6CF;
+	    padding-bottom: 200px;
+	    padding-top: 13px;
+	    width: 95%;
+		margin: 10px auto;
+	}
+		
+	
+	
+	h2 {
+		font-size:30px;
+	    padding: 13px 12px;
+	}
+	
+
+	
+	div.aboutme div {
+		text-align:center;
+		padding-left:10px;
+		
+	}
+	
+	div.aboutme p {
+		margin:10px;
+		font-size:20px;
+	}
+	
+	.portfolio {
+		justify-content:space-between;
+		border:2px black solid;
+		width: 95%;
+		margin: 10px auto;
+		background-color:white;
+	}
+	
+	.portfolio h2 {
+		padding: 0 23px;
```

Comment:
Would suggest adding a little bit of top and bottom padding, something like `padding: 10px 23px`.

This is optional though since it's a stylistic choice!